pkg/x/http/trace_test.go

```diff
@@ -0,0 +1,287 @@
+package http_test
+
+import (
+	"bytes"
+	"cmp"
+	"io"
+	"net/http"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	xhttp "github.com/aquasecurity/trivy/pkg/x/http"
+)
+
+func TestTraceTransport_RoundTrip(t *testing.T) {
+	// Disable colors for consistent testing
+	t.Setenv("NO_COLOR", "1")
```

Comment:
For my own understanding, how does this work? Is it part of the CLI framework which disables colour output if this environment variable is set?